pandas/core/dtypes/missing.py

```diff
@@ -156,7 +157,9 @@ def _isna(obj, inf_as_na: bool = False):
         raise NotImplementedError("isna is not defined for MultiIndex")
     elif isinstance(obj, type):
         return False
-    elif isinstance(obj, (ABCSeries, np.ndarray, ABCIndexClass, ABCExtensionArray)):
+    elif isinstance(
```

Comment:
excellent! can you include the asv's which hit this

Reply:
Done